govcd/metadata.go

```diff
@@ -123,8 +121,8 @@ func (vapp *VApp) AddMetadata(key string, value string) (Task, error) {
 	return addMetadata(vapp.client, types.MetadataStringValue, key, value, vapp.VApp.HREF)
 }
 
-// Adds metadata (type MetadataStringValue) to the vApp
-// The function supports passing a typedValue. Use one of the consts defined.
+// Adds metadata to the vApp
+// The function supports passing a typedValue. Use one of the constants defined.
```

Comment:
what constant?

Reply:
Four new constants have been added:
https://github.com/mikeletux/go-vcloud-director/blob/f134041848100263630e762d599f4a37457b3b14/types/v56/constants.go#L484-L490

Reply:
I meant to add a reference to them :)

Reply:
Done!